supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -192,16 +193,9 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     new JesExpressionFunctions(List(gcsFileSystem), callContext)
   }
 
-  private lazy val knownInputs = {
-    val unqualifiedWorkflowInputs = workflowDescriptor.inputs map {
-      case (fqn, v) => splitFqn(fqn)._2 -> v
-    }
-    unqualifiedWorkflowInputs ++ jobDescriptor.inputs
-  }
-
   private val lookup: ScopedLookupFunction = {
     val declarations = workflowDescriptor.workflowNamespace.workflow.declarations ++ call.task.declarations
-    WdlExpression.standardLookupFunction(knownInputs, declarations, callEngineFunctions)
+    WdlExpression.standardLookupFunction(jobDescriptor.inputs, declarations, callEngineFunctions)
```

Comment:
Short answer, `jobDescriptor.inputs` is all you should need. The use of `unqualifiedWorkflowInputs` is from a version prior to input evaluation refactor.
Now as we discussed yesterday, although working this whole thing is confusing and deserves a re-re-re-re-arrangement.